indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>

```

Comment:
```suggestion
<!Doctype html>
```
I think we will be needing an Uppercase for `Doctype` definitions

Reply:
The name of the index file in this scenario does not help to understand what that file is for. I feel this would work much better if it was named index.html and the website can render this by default when the user navigates to this folder in the server.